### PR TITLE
LTP: Updated ltp-open-posix to run specific group test.

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Make sure to run timers testcases from ltp-open-posix.
Parse the results and send it lava-test-case.

Signed-off-by: lakshmipathi.g <lakshmipathi.ganapathi@collabora.com>

### CODE REVIEW

File: automated/linux/ltp-open-posix/ltp-open-posix.sh [hunk 1]
```diff
@@ -103,4 +125,9 @@ else
     install_ltp_open_posix
 fi
 info_msg "Running run-ltp-open-posix"
-run_ltp_open_posix
+
+if [ "${GRP_TEST}" = "TMR" ]; then
```
Comment (gctucker) [inline automated/linux/ltp-open-posix/ltp-open-posix.sh]: Why are the timers tests treated differently? We should be able to run any group from the open POSIX test suite.  This patch should be reworked to make it more general and not specific to the timers tests.

**Review by gctucker — DISMISSED** [review]
A few issues with the commit message:

```
LTP: Updated ltp-open-posix to run specific group test.

Make sure to run timers testcases from ltp-open-posix.
Parse the results and send it lava-test-case.

Signed-off-by: lakshmipathi.g <lakshmipathi.ganapathi@collabora.com>
```

Please use the present tense, so "update ltp-open-posix" rather than "updated...".

It would be better to use your full name in the trailer: `Signed-off-by: lakshmipathi.g`

The patch should not be about timers tests in particular but any open POSIX tests.  See my other comments about this.

**Comment by Lakshmipathi** [comment]
https://lava.collabora.co.uk/scheduler/job/3468628

**Comment by gctucker** [comment]
@Lakshmipathi Looking fine, however the commit message still mentions timers explicitly even though this is a generic patch for Open POSIX tests:
```
Make sure to run timers testcases from ltp-open-posix.
Parse the results and send it lava-test-case.
```
Is this new version of the patch working as expected on staging?

**Comment by Lakshmipathi** [comment]
updated commit message. Seems like its still using older config - so job failing: https://storage.staging.kernelci.org/kernelci/staging-next/staging-next-20210408.0/x86_64/x86_64_defconfig+kernel-configs-x86-chromebook.config/gcc-8/lab-collabora/ltp-timers-hp-11A-G6-EE-grunt.html

**Comment by Lakshmipathi** [comment]
Sample Results from staging:
https://staging.kernelci.org/test/job/kernelci/branch/staging-mainline/kernel/staging-mainline-20210416.0/plan/ltp-timers/
https://staging.kernelci.org/test/plan/id/6078e0a20bfe3738d87747c2/
https://staging.kernelci.org/test/plan/id/6078df4f4ca8b9707a7747ef/

**Review by gctucker — APPROVED** [review]

**Comment by gctucker** [comment]
Merged so it will be on the kernelci.org branch, we'll have to keep doing a `git pull --rebase` until it's upstreamed.

**Pull request merged**